src/Aspire.Hosting.Elasticsearch/ElasticsearchBuilderExtensions.cs

```diff
@@ -44,6 +44,9 @@ public static class ElasticsearchBuilderExtensions
         IResourceBuilder<ParameterResource>? password = null,
         int? port = null)
     {
+        ArgumentNullException.ThrowIfNull(name);
+        ArgumentNullException.ThrowIfNull(builder);
+
```

Comment:
This is purely my opinion, but I would change the order so that there is a sequence similar to the incoming parameters. builder, name